project/build.scala

```diff
@@ -140,14 +140,14 @@ object build extends Build {
     , settings = standardSettings ++ lib("testing") ++ Seq[Settings](
       name := "mundane-testing-extra"
     ) ++ Seq[Settings](libraryDependencies ++= depend.specs2 ++ depend.specs2Extra)
-  ).dependsOn(testing)
+  ).dependsOn(testing % "test->test")
```

Comment:
Not that it matters, but that doesn't look right. Is `testing-extra` supposed to extend `testing` with more util-like functionality (and depend on it for `compile`).

In any case it doesn't look like `testing-extra` actually needs `testing` in the first place